src/statusnotifieritem/statusnotifieritem.cpp

```diff
@@ -62,17 +68,17 @@ StatusNotifierItem::StatusNotifierItem(QString id, QObject *parent)
 
 StatusNotifierItem::~StatusNotifierItem()
 {
-    QDBusConnection::sessionBus().unregisterObject("/StatusNotifierItem");
-    QDBusConnection::sessionBus().unregisterService(mService);
-    QDBusConnection::sessionBus().disconnectFromBus(mService);
+    mSessionBus.unregisterObject("/StatusNotifierItem");
```

Comment:
I'm also curious about that. @luis-pereira, what might happen without `QLatin1String`?